packages/react-dom/src/__tests__/ReactDOMConsoleErrorReporting-test.js

```diff
@@ -56,7 +56,8 @@ describe('ReactDOMConsoleErrorReporting', () => {
 
   describe('ReactDOMClient.createRoot', () => {
     it('logs errors during event handlers', async () => {
-      spyOnDevAndProd(console, 'error');
```

Comment:
I had to switch to manual mocking. The spyOnDevAndProd helper doesn't work in a bunch of cases and I have no idea why.

Reply:
I guess spyOn also calls the inner one. So in these cases it's because we're no longer ignoring uncaught error in logs.